shell/shared/admin.js

```diff
@@ -85,6 +94,7 @@ if (Meteor.isClient) {
       var handleErrorBound = handleError.bind(state);
       Meteor.call("setAccountSetting", "google", event.target.googleLogin.checked, handleErrorBound);
       Meteor.call("setAccountSetting", "github", event.target.githubLogin.checked, handleErrorBound);
+      Meteor.call("setAccountSetting", "emailToken", event.target.emailTokenLogin.checked, handleErrorBound);
```

Comment:
Let's prohibit enabling email login if no SMTP server is configured, with an appropriate error message.

Ideally we'd also test somehow that the SMTP server actually works, just to make things idiot-proof.